### PR TITLE
Add BOM

### PR DESCRIPTION
**Pull request by cdjackson**
@kaikreuzer can you check this please? I'm not completely sure if this is correct, but I think it's better to have a BOM here, and then just reference this BOM in the openhab-deps repo.

Signed-off-by: Chris Jackson <chris@cd-jackson.com>

### CODE REVIEW

File: bom/pom.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles<</groupId>
+    <artifactId>org.openhab.addons.zigbee.reactor/artifactId>
```
Comment (kaikreuzer) [inline bom/pom.xml]:
```suggestion
    <artifactId>org.openhab.addons.zigbee.reactor</artifactId>
```

Comment (kaikreuzer) [reply bom/pom.xml]: Please check yourself that the project at least builds locally - thanks!

**Comment by cdjackson** [comment]
Sorry - I've not managed to get it to build here, and as I said, I'm not really sure how this new build system works which is why I’ve been trying to get someone to help with this for the past few weeks.  If you have suggestions as to how make it work, then I’d really appreciate it.

I'll close this then - if someone who knows the system can update it, it would be appreciated.

**Comment by t-8ch** [comment]
Few notes:

The versions in the bom should not be properties, as these would be resolved in the importing project.
(maybe the maven release plugin would update these versions too. Or the flatten plugin)

The dependencies in the bom should be wrapped in `pluginManagement`.

In general is a BOM for this project not very useful, as all component share the same version, and can easily be managed with a maven property in the consumer.

**Comment by cdjackson** [comment]
Thanks - are you able to create a PR to get this resolved? It would be appreciated if you could.

**Pull request closed without merging**